.github/workflows/ci.yml

```diff
@@ -241,34 +241,30 @@ jobs:
               ./configure --enable-unicode --enable-werror
               gmake -k
 
-# This is broken since the ncursesw merge in NetBSD, cf.
-#  https://github.com/NetBSD/pkgsrc/commit/4f72c2c3a2081165a195b2667bbd5e2030397442
-# DL240722, disabling this until somebody with NetBSD can fix the build
-#
-#  build-netbsd-latest-gcc:
-#    runs-on: ubuntu-22.04
-#    timeout-minutes: 20
-#    steps:
-#      - uses: actions/checkout@v4
-#        with:
-#          submodules: recursive
-#      - name: Compile
-#        uses: vmactions/netbsd-vm@v1
-#        with:
-#          release: '10.0'
-#          usesh: true
-#          prepare: |
-#            PATH="/usr/pkg/sbin:/usr/pkg/bin:$PATH"
-#            PKG_PATH="https://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/10.0/All/"
-#            export PATH PKG_PATH
-#            /usr/sbin/pkg_add pkgin
-#            pkgin -y install autoconf automake libtool ncurses ncursesw gmake git
-#            git config --global --add safe.directory /home/runner/work/htop/htop
-#          run: |
-#            set -e
-#            ./autogen.sh
-#            CPPFLAGS="-I/usr/pkg/include" ./configure --enable-unicode --enable-werror
-#            gmake -k
+ build-netbsd-latest-gcc:
+     runs-on: ubuntu-22.04
+     timeout-minutes: 20
+     steps:
+       - uses: actions/checkout@v4
+         with:
+           submodules: recursive
+       - name: Compile
+         uses: vmactions/netbsd-vm@v1
+         with:
+           release: '10.0'
+           usesh: true
+           prepare: |
+             PATH="/usr/pkg/sbin:/usr/pkg/bin:$PATH"
+             PKG_PATH="https://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/10.0/All/"
+             export PATH PKG_PATH
+             /usr/sbin/pkg_add pkgin
+             pkgin -y install autoconf automake libtool ncurses ncursesw gmake git
+             git config --global --add safe.directory /home/runner/work/htop/htop
+           run: |
+             set -e
+             ./autogen.sh
+             CPPFLAGS="-I/usr/pkg/include -I/usr/pkg/include/ncurses" LDFLAGS="-L/usr/pkg/lib -Wl,-R/usr/pkg/lib" ./configure --enable-unicode --enable-werror
```

Comment:
Restore the build with `ncurses` support.

Reply:
```
  htop 3.4.0-dev

  platform:                  netbsd
  os-release file:           /etc/os-release
  (Linux) proc directory:    /proc
  (Linux) openvz:            no
  (Linux) vserver:           no
  (Linux) ancient vserver:   no
  (Linux) delay accounting:  no
  (Linux) sensors:           no
  (Linux) capabilities:      no
  unicode:                   yes
  affinity:                  no
  unwind:                    no
  hwloc:                     no
  debug:                     no
  static:                    no
```

Output of `./configure`

Reply:
One issue here I see is that you need explicit `CFLAGS` and `LDFLAGS` in order to link against `ncurces` and not `curses`. This is not convenient for builders, and is the issue I wish to see it resolved in #1506.

Reply:
When building via pkgsrc with the `ncurses` option these flags are set before doing `./configure`
- https://github.com/NetBSD/pkgsrc/blob/trunk/sysutils/htop/Makefile#L21

The `${MAKE_ENV}` sets them in this case.

I agree it is not clean but with #1506 may be we can tweak the build to look cleaner outside the pkgsrc build script.

Reply:
I have removed the `LDFLAGS` and I think it should work now.

Reply:
@fraggerfox Is it possible to obtain the library include path via a `pkg-config` command? That's the way I expect to see it done.

```text
LDFLAGS=$($PKG_CONFIG --libs ncurses)
```

Similarly for CPPFLAGS:

```text
CPPFLAGS=$($PKG_CONFIG --cflags ncurses)
```

Reply:
@Explorer09 : Should be done in the latest commit.